team-telnyx/webrtc
# feat(WEBRTC-187): investigate and fix bundle size

- Reduce bundle size of the projects SDK's by **33%**.
 - Update examples to use the last version of SDK's.

## 📝 To Do


## ✋ Manual testing

1. Navigate o `packages/js/examples/react-audio`
2. Run `npm i && npm setup`
3. Make a call and see if it is working.

## 🦊 Browser testing

### Desktop

- [x] React Native Android
